pointsource_imaging/MIRI_Aperture_Photometry_solution.ipynb

```diff
@@ -161,14 +161,27 @@
     "### 2.2<font color='white'>-</font>Plot configuration<a class=\"anchor\" id=\"plot\"></a> ###"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    %matplotlib notebook
</pre><p><code>%matplotlib notebook</code> actually didn't work for me in Lab, and I was able to find others who have reported the same. I now recommend <code>%matlotlib inline</code> as the default.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jwebbinar_prep/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
OK. I modified the line and left online `inline`. I also increased the figure size.

Reply:
In the solution version, I forgot to remove the comment above `%matlotlib inline`. I removed it from the live solution.

Reply:
I don't think the comment is a problem in its current form. I'll leave it up to you whether to put it back in the live version or also remove it from the solution version.

Reply:
If you can remove it from the solution, go ahead. If I need to do it, I will push a new version. The cell below the warning in Sect 2.2 should be the following:
`%matplotlib inline
# Figure size
plt.rcParams['figure.figsize'] = [15, 15]
# Figure text and font
plt.rc('text', usetex=True)
plt.rc('font', size=15)`

Reply:
I went ahead and removed the comment from the solution notebook.